krizten/Memoire-WebClient
# #165537378 Develop Settings Page UI

### What does this PR do?
- Build out settings page.
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the settings page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ```ft/165537378/settings-page```
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/app/settings```
#### What are the relevant pivotal tracker stories?
#165537378
#### Screenshots (if appropriate)
![settings-desktop](https://user-images.githubusercontent.com/29978455/58776602-4092f300-85c3-11e9-9dab-a6da8a25e212.png)
![settings-mobile](https://user-images.githubusercontent.com/29978455/58776604-4688d400-85c3-11e9-9852-84684f70226e.png)
